front/src/common/Map/WarpedMap/WarpedMap.tsx

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable no-console */
+import { useSelector } from 'react-redux';
+import React, { FC, PropsWithChildren, useEffect, useMemo, useState } from 'react';
```

Comment:
:warning: linter warning
```suggestion
import React, { FC, useEffect, useMemo, useState } from 'react';
```